packages/twenty-server/src/engine/api/graphql/graphql-query-runner/utils/check-string-is-database-event-action.ts

```diff
@@ -0,0 +1,7 @@
+import { DatabaseEventAction } from 'src/engine/api/graphql/graphql-query-runner/services/api-event-emitter.service';
+
+export const checkStringIsDatabaseEventAction = (value: string): boolean => {
```

Comment:
[I'm not expert enough but ](https://stackoverflow.com/questions/51124837/what-is-reason-to-use-is-operator-in-typescript)

check `isFieldDate` code